kit/typing.py

```diff
@@ -22,3 +22,15 @@ def __call__(self, func: _F) -> _F:
         super_method = getattr(self.interface, func.__name__, None)
         assert super_method is not None, f"'{func.__name__}' does not exist in {self.interface}"
         return func
+
+
+class parsable:  # pylint: disable=invalid-name
+    """Mark an object's __init__ as parsable by Configen, so only 3.7 type annotations should be used."""
```

Comment:
Something like this seems more accurate:

```suggestion
    """Mark an object's __init__ as parsable by Configen, so only pre-3.9 type annotations should be used."""
```